### DISCUSSION TITLE
GitHub Copilot General Availability in the CLI 🎉

### DISCUSSION THREAD

**Discussion by cgtechcommunity** [community/community#113558] [Copilot News and Announcements]
![CLI GA](https://github.com/community/community/assets/156237126/58059177-2dd4-4a6f-8714-903f6dc2997d)

We are excited to announce Copilot in the CLI is now generally available (GA) for all our Copilot Individual, Business, and Enterprise customers! 🚀 

Copilot in the CLI allows users to access the power of GitHub Copilot to get command suggestions and explanations without leaving the terminal.

See the full [Changelog post](https://github.blog/changelog/2024-03-21-github-copilot-general-availability-in-the-cli/) to learn how to get started.

Let us know your thoughts by reacting here or posting your feedback in the [Copilot public repository](https://github.com/github/gh-copilot). 🙏

**Comment by cgtechcommunity** [comment]
And here is an excellent [YouTube video](https://youtu.be/fHwtrOcLAnI?feature=shared) by @ladykerr explaining GitHub Copilot in the CLI for anyone who prefers learning through video. 🎉

**Comment by ramesh2051** [comment]
Hey @cgtechcommunity,

Even after enable access at the organizational level, the issue with the copilot CLI persists.


![image](https://github.com/community/community/assets/95218310/19a3ea73-99da-461b-9909-0cca8315c530)

Please read the error below for reference; it indicates that the user is not authorised to utilise the copilot capability. However, I have a copilot for a business license with the same organization.


![image](https://github.com/community/community/assets/95218310/b91fd441-221e-45d9-8969-b309d61f4c39)
 
Can you help on this at earliest..

**Reply by Akash1134** [reply]
Hi @ramesh2051 ,

With regard to this issue, please check for an email sent to your team on March 14th with the subject line `Subject: GitHub Copilot in the CLI will be generally available on March 21`.

If you do not have a billing option attached to your GitHub Copilot subscription you will need to set that up before following the below instructions.

In order to resolve this issue you should go to your [Enterprise Copilot settings](https://github.com/enterprises/ENTERPRISE_NAME/settings/copilot?tab=policies) Or at the [Organization level](https://github.com/organizations/ORG_NAME/settings/copilot/policies) and you would need to change the option Copilot in the Cli to enabled. This would fix the issue at this time according to our Copilot engineering team.

**Comment by aa659** [comment]
When going to enable Copilot in the CLI in the enterprise settings, it still gives a warning that additional data will be collected. Is this intentional (and if so, what additional data) or should that warning no longer appear as Copilot in the CLI is no longer a beta/pre-release feature?

![image](https://github.com/community/community/assets/135951391/260e11c7-cb75-40a0-8835-7527f792ae1e)
